app/Platform/EventServiceProvider.php

```diff
@@ -47,11 +48,13 @@ class EventServiceProvider extends ServiceProvider
         AchievementPublished::class => [
             DispatchUpdateGameMetricsJob::class, // dispatches GameMetricsUpdated
             DispatchUpdateDeveloperContributionYieldJob::class, // dispatches UpdateDeveloperContributionYield
+            UpdateTotalGamesCount::class,
```

Comment:
Eventually I see this also updating TotalAchievementsCount. Should that logic just be rolled into this class and renamed?

Reply:
Good point -- personally I could go either way on this (having two separate actions vs a single action).

I lean more towards having two separate actions just because the queries to construct the two data points are likely quite different, but that's a very very loosely-held opinion.